From fc64da46f22415c4c5af6b2541c66f0bed86e377 Mon Sep 17 00:00:00 2001
From: pramod kotreshappa <kpramod@codeaurora.org>
Date: Wed, 24 Jul 2019 15:03:52 -0700
Subject: [PATCH 28/38] Bluetooth: Fix for out of bound exception

Increase array length to accomadate TWS+ codec entry in
DevUI settings.

 Conflicts:
	packages/SettingsLib/res/values/arrays.xml

CRs-fixed: 2495783
Change-Id: I3b334f9420eff0850ead4db9d452c69ce4295c50
Signed-off-by: Volodymyr Zhdanov <wight554@gmail.com>
---
 packages/SettingsLib/res/values/arrays.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/SettingsLib/res/values/arrays.xml b/packages/SettingsLib/res/values/arrays.xml
index 54214c6a..2e83a3e6 100644
--- a/packages/SettingsLib/res/values/arrays.xml
+++ b/packages/SettingsLib/res/values/arrays.xml
@@ -167,6 +167,7 @@
         <item>3</item>
         <item>4</item>
         <item>5</item>
+        <item>6</item>
     </string-array>
 
     <!-- Summaries for Bluetooth Audio Codec selection preference. [CHAR LIMIT=50]-->
-- 
2.28.0

